extensions/wikia/TrackingOptIn/js/trackingOptIn.js

```diff
@@ -44,9 +60,54 @@ define('wikia.trackingOptIn', [
 		window.Wikia.consentQueue.push(callback);
 	}
 
+	function geoRequiresTrackingConsent() {
+		return geoRequiresConsent;
+	}
+
+	function getPrebidGlobalVendorList() {
```

Comment:
Do we really need to request this list on each pv? Does it really matter for us that this list is changing? We have hardcoded list of enabled providers (like `10,32,52,69,76`). Can we "cache"/hardcode it and use it as a module?

Reply:
We don't request it every PV because consent string is cached for 7 days, and then request is not being performed.